2-mandatory/6-budgets.js

```diff
@@ -1,18 +1,24 @@
-/**
-Get Sum of People's Budget
-Create the function that takes an array with objects and returns the sum of people's budgets.
-For example:
-getBudgets([
+
+// Get Sum of People's Budget
+// Create the function that takes an array with objects and returns the sum of people's budgets.
+// For example:
+peopleArray = [
   { name: "John",  age: 21, budget: 29000 },
   { name: "Steve",  age: 32, budget: 32000 },
   { name: "Martin",  age: 16, budget: 1600 }
-])
-Should give return the answer of 62600.
-**/
+]
+// Should give return the answer of 62600.
+
 
 function getBudgets(peopleArray) {
```

Comment:
nice. as in the previous weeks assignment here also reduce could be used. But I prefer this solution, this is just as note :)